pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -335,6 +335,16 @@ public void setSubscriptionExpirationTime(@PathParam("tenant") String tenant,
         internalSetSubscriptionExpirationTime(expirationTime);
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/deduplication")
```

Comment:
We already have policy rest API. The `deduplication` is a property in the policy. Why do we need to introduce a new Rest API for this?

Reply:
We do not yet have an interface to return the entire policies, but I don't think it is a good way to return the entire policies every time.

Reply:
We have the get namespace policy API: https://github.com/apache/pulsar/blob/7c6f5e2bbc8b753609b89a7589f480a57c711260/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java#L119

Reply:
The user can get entire policies, or only a single piece of policies, there is no conflict

Reply:
The documentation we give to users does not say that every interface returns all the data. Although we can handle it in the SDK, the user may directly call the API.
So I think it is better to open the interface separately.